airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/ConnectorMetadataDefinitionV0.py

```diff
@@ -148,6 +148,10 @@ class Data(BaseModel):
         "api", "database", "file", "custom", "message_queue", "unknown"
     ]
     releaseStage: Literal["alpha", "beta", "generally_available", "source"]
+    tags: Optional[List[str]] = Field(
```

Comment:
Are there required tags?  At least maybe "language:*"?

Reply:
Evan you reviewed so fast! Thank you!

I put this up slightly prematurely as I was writing the mandatory language check.

Thats now up in a new commit with tests